modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/indexer/test/UserIndexerTest.java

```diff
@@ -856,6 +855,45 @@ protected void toggleCreateSite(Organization organization, boolean site)
 	@Inject
 	protected UserLocalService userLocalService;
 
+	private Version _getElasticsearchVersion() {
+		ConnectionInformation connectionInformation =
+			_searchEngineInformation.getConnectionInformationList(
+			).get(
+				0
+			);
+
+		NodeInformation nodeInformation =
+			connectionInformation.getNodeInformationList(
+			).get(
+				0
+			);
+
+		return Version.parseVersion(nodeInformation.getVersion());
+	}
+
+	private String _getExpectedFullNameHighlight(
+		String firstName, String middleName, String lastName) {
+
+		String expectedTitleHighlight = StringBundler.concat(
+			HighlightUtil.HIGHLIGHT_TAG_OPEN, firstName,
+			HighlightUtil.HIGHLIGHT_TAG_CLOSE, StringPool.SPACE, middleName,
+			StringPool.SPACE, HighlightUtil.HIGHLIGHT_TAG_OPEN, lastName,
+			HighlightUtil.HIGHLIGHT_TAG_CLOSE);
```

Comment:
You can inline it. Do not need to create a variable for this. See https://github.com/AlmirFe/liferay-portal/commit/9648821efa67586db3e2756a7e81fd70bb2ed825

Reply:
Thx Almir! I will Cherry pick it